source/Halibut.Tests/Transport/AuthorizedTcpConnectionsLimiterFixture.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright 2012-2013 Octopus Deploy Pty. Ltd.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
nit: Does this need to be in the file? Does your ide just always add it?

Reply:
the IDE always added it 🤷

Reply:
I can remove it.. Not sure why it added it